knowledge-base/scheduler-conditionally-disable-drag-for-some-appointments.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Conditionally Disable Dragging for Specific Scheduler Appointments
+description: Learn how to disable the drag functionality for certain appointments in the Scheduler component based on a condition.
+type: how-to
+page_title: How to Conditionally Disable Drag for Certain Appointments in Blazor Scheduler
+slug: scheduler-kb-conditionally-disable-drag-for-some-appointments
+tags: scheduler, blazor, drag, disable, appointment
+res_type: kb
+ticketid: 1660283
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>Scheduler for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+I want to disable the dragging of some appointments based on a condition, such as the status of the appointment. 
+
+This KB article also answers the following questions:
+- How can I prevent specific appointments from being dragged in the Scheduler?
+- Is it possible to conditionally disable drag functionality for appointments in the Scheduler?
+- How to use CSS and event handling to disable dragging for certain appointments?
+
+## Solution
+
+To conditionally disable the drag of an appointment based on your desired condition, follow these steps:
+
+1. Handle the [OnItemRender event]({%slug scheduler-events%}#itemrender) to add a custom CSS class to the appointments that you want to disable the dragging for.
```

Comment:
"... add a custom CSS class to the non-draggable appointments" is shorter. The remove keywords are used plenty of times elsewhere, so it's not a problem.